test/specs/lobbyAdmitParticipant.js

```diff
@@ -0,0 +1,46 @@
+import openSession from "../helpers/lobbyHelper"
+import { BASE_URL } from "../helpers/constants"
+
+import openParticipantsPane from "../helpers/openParticipantsPane"
+
+describe('Activate lobby and admit participant', () => {
+    it('should open jitsi-meet app, enable lobby and admit participant', async () => {
+        let roomName;
+        const capabilities = await browser.requestedCapabilities;
+        switch (capabilities.browserName) {
+            case 'chrome':
+                roomName = 'ChromeRoomNameTest'
+                break;
+            case 'firefox':
+                roomName = 'FirefoxRoomNameTest'
+                break;
+            default:
+                roomName = 'SafariRoomNameTest'
+        }
+        await openSession({
+            moderator: true,
```

Comment:
Ah I see now. You don't need this, the first participant will the the moderator, and since you are waiting for it...

Reply:
This has changed with a new approach